src/gui/torrentmodel.h

```diff
@@ -33,15 +34,20 @@
 #define TORRENTMODEL_H
 
 #include <QAbstractListModel>
+#include <QColor>
+#include <QIcon>
+#include <QPixmap>
 #include <QList>
 
+#include "base/bittorrent/torrenthandle.h"
```

Comment:
So that `BitTorrent::TorrentState` is a complete type. But I could get around that by the parameter instead being `const BitTorrent::TorrentState &` - then a forward declaration would be enough.